host_core/lib/host_core/wasmcloud/runtime_server.ex

```diff
@@ -122,13 +125,24 @@ defmodule HostCore.WasmCloud.Runtime.Server do
 
   @impl true
   def handle_info(
-        {:invoke_callback, claims, binding, namespace, operation, payload, token},
+        {:invoke_callback, claims, {binding, namespace, operation}, payload, call_context, token},
```

Comment:
Note that changing this to a tuple was purely to avoid a Rustler issue where there's a macro that doesn't seem to be implemented for 8 arguments. Happy to change the tuple format if desired

Reply:
I like this and it's a pinch more idiomatic Elixir to bundle up args into a logically grouped tuple